ext/pdo_pgsql/tests/result_memory_size.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+PDO PgSQL PDOStatement::getAttribute(PDO::PGSQL_ATTR_RESULT_MEMORY_SIZE)
+--EXTENSIONS--
+pdo
```

Comment:
This statement is unnecessary because pdo is always loaded when pdo_pgsql is loaded.

Reply:
Thanks, I fixed: https://github.com/php/php-src/pull/14260/commits/8701022ad1b113937bb3e58c6b3f5dc358bb24c1